src/core/Job.ts

```diff
@@ -166,42 +173,67 @@ export default class Job {
                     return filteredLocations.length > 0;
                 });
 
+            // Job post region is not in our list
+            const isRegionUnknown = !this.config.regionNames.find((location) => {
+                return post.location.match(new RegExp(location, "i"))
```

Comment:
I wonder if this could lead to unexpected matches such as "British Columbia" matching with "British"?

Reply:
Rarely, but not impossible. Agree that we could be more strict here

Reply:
It's doing the same RegExp as the one above it, but in reverse no? So is there any chance of locations matching both?

Reply:
Actually there was a bigger problem: `regionNames` wasn't what I expected. It was working but for the wrong reasons.

I think the main reason we're using regex is because the location as scrapped from GH was surrounded in parenthesis: "(Home based - London)". If we get rid of them we can check for string equality which is simpler IMO